apps/main/src/dex/lib/networks.ts

```diff
@@ -351,7 +351,8 @@ export async function getNetworks() {
         stableswapFactory: true,
         twocryptoFactory: true,
         tricryptoFactory: true,
-        isLite: true,
+        pricesApi: chainId == 146, // only enabled for sonic
+        isLite: chainId != 146, // only disable for sonic
```

Comment:
imo it's more helpful to explain why not what
```suggestion
        pricesApi: chainId == 146, // sonic is upgraded from lite to full
        isLite: chainId != 146, // sonic is upgraded from lite to full
```